data/merchant intro.txt

```diff
@@ -0,0 +1,138 @@
+	destination "New Switzerland"
+	npc
+		government "Pirate"
+		personality entering plunders
+		fleet "Bounty Hunters"
+	on offer
+		conversation
+			`"Captain," salutes the man, "This job involves <bunks> travelers trying to get to <destination> with <cargo> now, I have no idea why they need so much of it, but, get it over there alright?"`
+			choice	
+				`	"Yes, sir."`
+					accept
+				`	"Sorry, but I can't."`
+					decline
+	on complete
+		payment 50000
+		conversation
+			`You help the travelers lug his cargo away, still wondering why so much of it is needed. You see the first man walking up to you. "Well, Captain, may I ask you for your name?"`
+			choice
+				`	"It's <first> <last>."`
+					goto name
+				`	"I would rather not say."`
+					goto anonymous
+			label name
+			`	"Thank you, <first>"
+				goto next
+			label anonymous
+			`	The man smiles slightly. "That's perfectly fine, Captain <last>," he says, surprising you, "just do a better job of covering your pilot's license next time."`
+			label next
+			`	"Anyways, if you're still interested, meet me in the spaceport afterwards so we can converse further."`
+
+mission "Merchant Start 4"
+	name "Fast delivery"
+	description "Get <bunks> doctors to with <cargo> to <destination>
+	passengers 5
+	cargo "medical supplies" 15
+	deadline 13
+	to offer
+		has "Merchant Start 3: done"
+	source "New Switzerland"
+	destination "Moonshake"
+	npc
+		government "Pirate"
+		personality entering plunders
+		fleet "Bounty Hunters"
+		fleet "pirate raid"
+	on offer
+		conversation
+			`You find the man quietly sitting at a table drinking a cup of coffee.  He waves you over and signals to the barman to bring another drink. "Thank you for joining me, Captain <last>."
+			choice
+				`	"It's my pleasure."`
+					goto pleasantries
+				`	What's the mission?"
+					goto point
+			label pleasantries
+			`	He smiles, "Politeness can bring you places. There aren't many of your type in this galaxy."`
+				goto mission
+			label point
+			`	He smiles, "Straight to the point are we?"
+			label mission
+			`	"Well, anyways, <destination> is in desperate need of <cargo> and doctors. You see, they are facing an epidemic there which started a week ago. Willing captains have been asked to go there as fast as you can or at the very least, before <date>." He winces, "The death toll is currently at 50,000 and rising. So I ask you, accept this job and get the supplies there as soon as possible."
+			choice
+				`	"I'll do it."`
+					goto thanks
+				`	"Sorry, I can't."`
+			`	"I might be wrong about you," he says, disappointment on his face as he leaves.`
+				decline
+			label thanks
+			`	"Thank you, I had a hunch you are the person I'm looking for!" He says before adding, "I almost forgot, because of the escalation in tensions between the Free Worlds and the Republic, the navy has pooled their resources to the southern planets, so expect to see an increase in pirate activity."`
```

Comment:
`Thank you` is an independent clause, so it needs a full stop after it. Also, "I had a hunch you _are_ the person" would sound more natural if it were "I had a hunch you _were_ the person I'm looking for", IMO.

Reply:
Maybe change it to you're?